### ISSUE TITLE
Unofficial guide to install wumiibo as current documentation is out of date as of 11/11/2023

### ISSUE THREAD

**Issue by Sweatydude** [hax0kartik/wumiibo#87]
### For use with Luma 3ds V13 and current release of wumiibo V4.2 seventh release
**Please use these steps instead of Install Wumiibo**
(_Assuming you have CFW installed, otherwise use google)_
Step 1: Insert your **SD card** into your computer, and download the **wumiibo.7z** file and unzip it
Step 2: Copy **wumiibohelper.cia** to any folder or to /luma/cias on the **SD card**
Step 3: Copy 0004013000004002.cxi to /luma/sysmodules _(you may need to create a new folder and rename it sysmodules if missing)_
Step 4: Navigate to /luma/titles _(you may need to create a new folder and rename it tiles if missing)_ and create a folder named: 0004013000004002
Step 5: Reinsert your SD card back into your 3DS and **hold select while powering on your 3DS**
Step 6: Once in Luma menu make sure you enable **both:** Enable game patching  **AND**  Enable loading external FIRMs and modules _also_ check that splash reads as: Splash: Off ( ) Before (x) After ( ) Payloads - If not Change that
Step 7: Once booted to your 3DS Home screen **open** FBI, it should open a file explorer, **navigate** to the folder with wumiibohelper.cia **from step 2** _(hopefully you created that /cias folder for organisation)_ and **install and keep CIA files**
Step 8: exit out of FBI and you should have new software on your home menu called Wumiibo, **choose** the option to generate amiibo and select the amiibo you want, exit out and **select** the last option to enable amiibo emulation which should reboot your 3DS
Step 9: Open the game you want to emulate amiibo for; and once prompted for amiibo to be scanned **press**: L + Down D Pad + Start
Step 10: Enjoy as your should get a new splash screen overlaying your game asking to select an amiibo to emulate

I created this because I found the documentation hard to follow as its not formatted and had out of date information and was especially hard to follow when your trying to explain the steps for someone over the phone (it might also assume you've been around the block when it comes to CFW and homebrew which I am not)

Hope this helps whoever is reading this and big thank you to the creators of Wumiibo: [hax0kartik] (https://github.com/hax0kartik) and to the user [ItsCrocoSwine] (https://github.com/ItsCrocoSwine) for commenting in this same issues section that you dont need to make the folder 0004013000004002 in /luma/titles but to instead copy 0004013000004002.cxi into /luma/sysmodules after the latest release prompting me to create an account and post this lol.

**Comment by hax0kartik** [comment]
This.. <https://github.com/hax0kartik/wumiibo/wiki/Usage-guide-for-Wumiibo-and-WumiiboHelper> is outdated?

**Comment by jevanstokely** [comment]
Thank you for explaining a manual install and the issues it can have. more helpful than official guide.

**Comment by Gengisclan** [comment]
Je vais tester et je te dis si ça marche
